Camel Quarkus 2.0.0 release announcement

ppalaga opened the pull request.

aldettinger commented on content/blog/2021/06/camel-quarkus-release-2.0.0/index.md:
```diff
@@ -0,0 +1,92 @@
+You have to press `r` to enable it.
+You may want to watch [this video](https://www.youtube.com/watch?v=rUyiTzbezjw) to learn more about the new continuous testing feature.
+
+Please refer to [Quarkus 2.0.0.Final announcement](https://quarkus.io/blog/quarkus-2-0-0-final-released/) for more details.
```
Its 404 today, but I guess it should be fine.

ppalaga replied:
Yeah, it is blocked by this one: https://github.com/quarkusio/quarkusio.github.io/pull/1037

aldettinger commented on content/blog/2021/06/camel-quarkus-release-2.0.0/index.md:
```diff
@@ -0,0 +1,92 @@
+* [DigitalOcean](/camel-quarkus/latest/reference/extensions/digitalocean.html)
+* [OpenStack](/camel-quarkus/latest/reference/extensions/openstack.html)
+* [XChange](/camel-quarkus/latest/reference/extensions/xchange.html)
+* [XQuery](/camel-quarkus/latest/reference/extensions/xchange.html)
```
Maybe it's xquery.html ?

ppalaga replied:
Thanks for noticing, it is saxon.html

aldettinger commented on content/blog/2021/06/camel-quarkus-release-2.0.0/index.md:
```diff
@@ -0,0 +1,92 @@
+In this release, we invested a lot of effort into reviewing existing tests and adding new test cases.
+We proceeded methodically, following the main Camel documentation.
+We took care to cover every use case mentioned on the Camel component pages.
+For example for the HTTP component, we went through the use cases mentioned on its component page](/components/latest/http-component.htm),
```
Should it be http-component.html ?

zbendhiba replied:
and there is a missing "[" for this link. Look at the preview

ppalaga replied:
+1, thanks!

aldettinger commented on content/blog/2021/06/camel-quarkus-release-2.0.0/index.md:
```diff
@@ -0,0 +1,92 @@
+Fixed issues:
+
+* [Milestone 2.0.0-M1](https://github.com/apache/camel-quarkus/milestone/13?closed=1)
+* [Milestone 2.0.0-M2](https://github.com/apache/camel-quarkus/milestone/13?closed=1)
```
miestone/16  for M2 ?

ppalaga replied:
Yep, thanks, another copy and paste mistake

aldettinger reviewed (commented):
Few typos proposal. Beyond, it' look good to me. Well done @ppalaga :+1:

ppalaga commented:
3d7f81e:

* Addressed the comments of @aldettinger 
* Added a paragraph about the extent of the test expansion effort https://github.com/apache/camel-website/pull/596/files#diff-ea0bf99904cb0b8bac8dfa648281b0d6b160e7e63cf67fab037cdf640924e617R73

Still blocked by https://github.com/quarkusio/quarkusio.github.io/pull/1037

ppalaga commented:
ee3e19f: fixed @zbendhiba 's "and there is a missing "[" for this link. Look at the preview"

ppalaga commented:
OK, the blocking PR was merged and https://quarkus.io/blog/quarkus-2-0-0-final-released/ is available
